src/main/java/lebron/Event.java

```diff
@@ -0,0 +1,73 @@
+package lebron;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that is an event with a start and end date. An Event task
+ * includes a description, start date, and end date.
+ */
+public class Event extends Task {
+
+    protected LocalDate start;
+    protected LocalDate end;
+
+    /**
+     * Constructs an Event task with the specified description, start date, and
+     * end date.
+     *
+     * @param description The description of the event.
+     * @param start The start date of the event.
+     * @param end The end date of the event.
+     */
+    public Event(String description, LocalDate start, LocalDate end) {
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
+
+    /**
+     * Returns a string representation of the event task. The string includes
+     * the task type, status icon, description, and formatted start and end
+     * dates.
+     *
+     * @return A string representing the event task.
```

Comment:
Maybe can consider including an example of how the string representation might look like?

Reply:
Aight Cheers Bro!